app/components/check-box.js

```diff
@@ -0,0 +1,28 @@
+import Ember from "ember";
+
+export default Ember.Component.extend({
```

Comment:
do we not have a checkbox component already ? :(

Reply:
no, it was not there